packages/address-consts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopify/address-consts",
-  "version": "1.0.5",
+  "version": "1.1.5",
```

Comment:
Did you bump this manually? We generally bump versions on a release, https://github.com/Shopify/quilt/blob/master/documentation/guides/release-and-deploy.md#3-versioning-and-tagging

Reply:
Also, it would be `v1.1.0`

Reply:
> Did you bump this manually? We generally bump versions on a release, https://github.com/Shopify/quilt/blob/master/documentation/guides/release-and-deploy.md#3-versioning-and-tagging

Yes I did. Okay thanks for the heads up. will catch up on the guides. 

> Also, it would be v1.1.0

😞 right. sorry totally slipped my mind for some reason.